src/content/docs/it/guides/backend/xata.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Xata & Astro
+description: Aggiungi un database serverless con ricerca full-text al tuo progetto con Xata
+type: backend
+service: Xata
+stub: false
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+[Xata](https://xata.io) è una **Piattaforma dati serverless** che combina le funzionalità di un database relazionale, un motore di ricerca e un motore di analisi, e li rende utilizzabili attraverso una singola REST API.
+
+## Aggiunta di un database con Xata
+
+### Prerequisiti
+
+- Un account [Xata](https://app.xata.io/signin) con un database. (Puoi utilizzare il database di esempio dall'interfaccia utente.)
+- Un Token di Accesso (`XATA_TOKEN_API`).
+- L'URL del tuo Database.
+
+Dopo aver aggiornato e inizializzato la [Xata CLI](https://xata.io/docs/getting-started/installation), avrai il tuo token per l'API nel file `.env` e l'URL del database.
+
+Alla fine della configurazione, dovresti avere:
+
+```ini title=".env"
+XATA_API_KEY=chiave_hash
+
+# Branch Xata che verrà utilizzato
+# se non esiste un branch xata con
+# lo stesso nome del tuo branch git
+XATA_FALLBACK_BRANCH=main
+```
+
+E l'URL del database:
+
+```ini title=".xatarc"
+{
+  "databaseUrl": "https://il-tuo-url-del-database"
+}
+```
+
+### Configurazione dell'ambiente
+
+Per avere IntelliSense e dei tipi per le tue variabili d'ambiente, modifica o crea il file `env.d.ts` nella tua cartella `src/`:
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly XATA_API_KEY: string;
+  readonly XATA_FALLBACK_BRANCH?: string;
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv;
+}
+```
+
+:::tip
+Per ulteriori informazioni sulle [variabili d'ambiente](/it/guides/environment-variables/) e sui file `.env` in Astro, leggi di più.
```

Comment:
"leggi di più" at the end of the sentence doesn't sound right to me. Maybe "Leggi questa guida per avere ulteriori informazioni [...]"?
What do you think?

Reply:
I agree

Reply:
I agree